src/constants/constants.js

```diff
@@ -1,4 +1,4 @@
 export const POLYGON_BRIDGE_ADDRESS =
   "0xFa3b46D79e8cA79dDA9611382a15C2775823705f";
 export const GODWOKEN_NFTS_ADDRESS =
-  "0x53cd4714E2F547c980684edF545f37fa344A4A30";
+  "0x5e72E3cE0cbC0975456c1C9D4314dCB2fd0a8Ce6";
```

Comment:
Why did the address change?

Reply:
Oh wait cause they are now deployed on mainnet

Reply:
yes!